src/docs/Glossary.adoc

```diff
@@ -67,13 +67,13 @@ Third Party Notification:: A <<_glossary_notification_methods,*Notification Meth
 Notification Method:: Element of a <<_glossary_notification_subscription,*Notification Subscription*>> which defines how a user is notified (portal, email, third-party, etc.).
 
 [[_glossary_cloud_automation_service]]
-Cloud Automation Service:: ProActive component that allows a user to easily manage any On-Demand Services (PaaS, IaaS and MaaS) with full Life-Cycle Management (create, deploy, pause, resume and terminate).
+Service Automation:: ProActive component that allows a user to easily manage any On-Demand Services (PaaS, IaaS and MaaS) with full Life-Cycle Management (create, deploy, pause, resume and terminate).
 
 [[_glossary_service_automation_portal]]
-Service Automation Portal:: Web interface of the <<_glossary_cloud_automation_service,*Cloud Automation service*>>.
+Service Automation Portal:: Web interface of the <<_glossary_cloud_automation_service,*Service Automation*>>.
 
 [[_glossary_workflow_automation_portal]]
-Workflow Automation Portal:: Web interface which provides an end-user portal to submit workflows, monitor their activity and receive workflow results.
+Workflow Execution Portal:: Web interface which provides an end-user portal to submit workflows, monitor their activity and receive workflow results.
```

Comment:
This definition is not fit for Workflow Execution because, contrarily to Service Automation which was just a renaming of Cloud Automation, Workflow Execution is a new portal that was made from scratch to replace Workflow Automation.
Please put this new definition that we used in a recent training:
"Workflow Execution portal, a web interface, is the main portal of ProActive Workflows & Scheduling and the entry point for end-users to submit workflows manually, monitor their executions and access job outputs, results, services endpoints, etc."